### PR TITLE
Nd bounding box intersected_with doc

### PR DESCRIPTION
**Pull request by xfolka**
### Description:
- Clarified the documentation for intersected_with method in nd_bounding_box

### Issues:
- fixes #...

### Todos:
Make sure to delete unnecessary points or to check all before merging:
 - [x] Updated Changelog
 - [x] Updated Documentation

### CODE REVIEW

File: webknossos/webknossos/geometry/nd_bounding_box.py [hunk 1]
```diff
@@ -554,12 +554,13 @@ def padded_with_margins(
...
-
-        If dont_assert is set to False, this method may return empty bounding boxes (size == (0, 0, 0))
+        If there is no intersection (resulting bounding box is empty) and dont_assert is False (default)
+        the function asserts. 
+        If dont_assert is set to True this method returns an empty bounding box if there is no intersection.  
 
         Args:
             other (NDBoundingBox): The other bounding box to intersect with.
-            dont_assert (bool): If True, the method may return empty bounding boxes.
+            dont_assert (bool): If True, the method may return empty bounding boxes. Default is False.
```
Comment (hotzenklotz) [inline webknossos/webknossos/geometry/nd_bounding_box.py:563]: It only return s a single BB, right? Therefore, the use of plural is not correct here.
```suggestion
            dont_assert (bool): If True, the method may return an empty bounding box. Default is False.
```

File: webknossos/webknossos/geometry/nd_bounding_box.py [hunk 2]
```diff
@@ -554,12 +554,13 @@ def padded_with_margins(
     def intersected_with(self: _T, other: _T, dont_assert: bool = False) -> _T:
         """
         Returns the intersection of two bounding boxes.
-
-        If dont_assert is set to False, this method may return empty bounding boxes (size == (0, 0, 0))
+        If there is no intersection (resulting bounding box is empty) and dont_assert is False (default)
+        the function asserts.
```
Comment (hotzenklotz) [inline webknossos/webknossos/geometry/nd_bounding_box.py:558]:
```suggestion
        the function raises an assertion error. 
```

**Review by hotzenklotz — CHANGES_REQUESTED** [review]
Thanks for the contribution. Please see my comments.

**Comment by hotzenklotz** [comment]
According to the CI you need to properly format your changes to pass. Please, run `./format.sh` for automatic formatting with `ruff`.

File: webknossos/webknossos/geometry/nd_bounding_box.py (hunk 1 above)
Comment (xfolka) [reply webknossos/webknossos/geometry/nd_bounding_box.py]: suggestions are implemented

**Review by hotzenklotz — APPROVED** [review]

**Pull request merged**